frontend-student/src/components/allcourses/CourseNew.jsx

```
@@ -0,0 +1,94 @@
+    fetchCourses();
+  }, []);
+
+  const getThumbnail = (thumbnail) => {
+    return thumbnail === "str" ? noimage : thumbnail;
+  };
+
+  const renderContent = () => {
+    if (loading) {
+      return (
+        <div className="flex justify-center my-10">
+          <div className="animate-spin rounded-full h-10 w-10 border-t-2 border-primary"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)